supporting_content/dimensions/pricingmodel.md

```diff
@@ -0,0 +1,33 @@
+# Column: PricingModel - WIP
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                 | Column                    |  Example Values  |
+|----------|--------------------------|---------------------------|------------------|
+| AWS | CUR                      |  product/PurchaseOption | On-Demand, Reserved Instances, Spot Instances, Dedicated Hosts |
+| Google Cloud | BigQuery Billing Export |              |  |
+| Microsoft | Cost Details             | PricingModel             | OnDemand, Spot, Reservation, SavingsPlan |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider | Data set                 | PricingModel     |      |
+|----------|--------------------------|----------------------------|------------------------------------------|
+| AWS | CUR ()                   |                 |              |
+| AWS | CUR ()                   |                |                    |
+| Google Cloud | BigQuery Billing Export |         |                            |
+| Google Cloud | BigQuery Billing Export |         |                               |
+| Microsoft | Cost Details (PricingModel)| Reservation                     | Savings Plan                     |
+| Microsoft | Cost Details (PricingModel)| Reservation                    | Reservation                      |
+| Microsoft | Cost Details (PricingModel)| Spot                    | Reservation                      |
+| Microsoft | Cost Details (PricingModel)| OnDemand                    | Reservation                      |
+
+## Documentation
+- AWS: https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html
+- GCP
+- Microsoft:  Understand usage details fields: https://learn.microsoft.com/azure/cost-management-billing/automate/understand-usage-details-fields
+
+## Discussion Topics
```

Comment:
Seems like we've had a lot of valuable conversations on this that should be captured here some how? I know there's been lots of twists and turns here but at least what we can recall should be captured so anyone wants to pick this up in the future can figure out why we went the direction we did.

Reply:
We need a Note taking bot to give us the transcripts and summary!

Reply:
Ya know, Teams has that built-in 😉

Reply:
+1 for Teams ;)